core/operation/src/main/java/uk/gov/gchq/gaffer/operation/impl/Limit.java

```diff
@@ -38,6 +38,7 @@
     @Required
     protected Integer resultLimit;
     private Iterable<? extends T> input;
+    private Boolean truncate = true;
```

Comment:
This could be a boolean instead of Boolean. Then in the handler you don't need to do the null check.